src/main/webapp/content/scss/global.scss

```diff
@@ -1220,3 +1220,29 @@ Note: This class assumes that the scrollbar is already present (e.g. overflow-y
         background-color: var(--scrollbar-color);
     }
 }
+
+.video-js {
+    width: 100% !important;
+    height: auto !important;
+    position: relative;
+}
+
+.video-js .vjs-control-bar {
+    position: absolute !important;
+    bottom: 0;
+    width: 100%;
+    z-index: 10;
+}
+
+.video-js.vjs-fullscreen {
+    position: fixed !important;
+    top: 0 !important;
+    left: 0 !important;
+    width: 100% !important;
+    height: 100% !important;
+    z-index: 9999 !important;
+}
+
+.video-js.vjs-fluid {
+    padding-top: 56.25%; // Aspect ratio 16:9
+}
```

Comment:
Can we declare this in the component file instead? (using ng-deep or by not sandboxing the CSS of the component)